python_modules/dagster/dagster/_cli/api.py

```diff
@@ -79,6 +84,43 @@ def send_to_buffer(event):
                 sys.exit(return_code)
 
 
+def _truthy_tag_value(dagster_run: DagsterRun, tag: str, default: str = "false") -> bool:
+    return dagster_run.tags.get(tag, default).casefold() not in ["false", "0", "off", "no", ""]
+
+
+def _should_start_metrics_thread(dagster_run: DagsterRun) -> bool:
+    return _truthy_tag_value(dagster_run, "dagster/run_metrics")
+
+
+def _enable_python_runtime_metrics(dagster_run: DagsterRun) -> bool:
+    return _truthy_tag_value(dagster_run, "dagster/python_runtime_metrics")
+
+
+def _report_container_metrics_as_engine_events(dagster_run: DagsterRun) -> bool:
+    return _truthy_tag_value(dagster_run, "dagster/container_metrics_engine_events")
+
+
+def _metrics_polling_interval(
+    dagster_run: DagsterRun, logger: Optional[logging.Logger] = None
+) -> float:
+    try:
+        return float(
+            dagster_run.tags.get(
+                "dagster/run_metrics_polling_interval_seconds",
+                DEFAULT_RUN_METRICS_POLL_INTERVAL_SECONDS,
+            )
+        )
+    except ValueError:
+        if logger:
+            logger.warning(
+                (
+                    "Invalid value for dagster/run_metrics_polling_interval_seconds tag."
+                    f"Setting metric polling interval to default value: {DEFAULT_RUN_METRICS_POLL_INTERVAL_SECONDS}."
```

Comment:
i would be inclined to disable the polling entirely if an incorrect value is passed in vs. falling back tot he default

Reply:
i think if the user has gone through the trouble of setting both tags on the run, he did expect to collect metrics. 

goes along the same argument for loose vs strict truthy evaluation. is it worth being annoying to be overly cautious on this?